OrbitCore/LinuxPerfRingBuffer.h

```diff
@@ -5,7 +5,9 @@
 //-----------------------------------
 // Author: Florian Kuebler
 //-----------------------------------
-#pragma once
+
+#ifndef ORBIT_CORE_PERF_RING_BUFFER_H_
+#define ORBIT_CORE_PERF_RING_BUFFER_H_
```

Comment:
I am actually not sure we want to get rid of 'pragme once', seems to be more convenient than ifndef/define...

Reply:
We are following the C++ styleguide with this as #pragma once is not part of the standard.